### PR TITLE
[Shop][Account] Order details on my account page

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Behat/Context/Ui/Shop/AccountContext.php [hunk 1]
```diff
@@ -12,12 +12,13 @@
 namespace Sylius\Behat\Context\Ui\Shop;
 
 use Behat\Behat\Context\Context;
+use Behat\Behat\Tester\Exception\PendingException;
```
Comment (tuka217) [inline src/Sylius/Behat/Context/Ui/Shop/AccountContext.php]: Should be removed

File: src/Sylius/Behat/Context/Transform/OrderContext.php [hunk 2]
```diff
@@ -38,10 +38,22 @@ public function __construct(OrderRepositoryInterface $orderRepository)
...
+
+    /**
+     * @Transform :orderNumber
+     * @Transform /^an order "([^"]+)"$/
+     */
+    public function getOrderNumber($orderNumber)
+    {
+        $orderNumber = str_replace('#', '', $orderNumber);
+
+        return $orderNumber;
```
Comment (tuka217) [inline src/Sylius/Behat/Context/Transform/OrderContext.php]: Shouldn't return int?

File: src/Sylius/Behat/Page/Shop/Account/Order/ShowPage.php [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+     * @return bool
+     */
+    private function hasAddress($elementText, $customerName, $street, $postcode, $city, $countryName)
+    {
+        return
+            (stripos($elementText, $customerName) !== false) &&
+            (stripos($elementText, $street) !== false) &&
+            (stripos($elementText, $city) !== false) &&
+            (stripos($elementText, $countryName.' '.$postcode) !== false)
+            ;
```
Comment (tuka217) [inline src/Sylius/Behat/Page/Shop/Account/Order/ShowPage.php]: Wrong indent

File: src/Sylius/Behat/Context/Ui/Shop/AccountContext.php [hunk 4]
```diff
@@ -285,6 +293,95 @@ public function thisOrderShouldHaveNumber(OrderInterface $order)
...
+    {
+        Assert::same(
+            $this->orderShowPage->getNumber(),
+            $orderNumber,
+            'The number of an order is %s, but should be %s.'
+        );
+    }
+
+    /**
+     * @Then I should see :customerName, :street, :postcode, :city, :countryName as shiping address
```
Comment (tuka217) [inline src/Sylius/Behat/Context/Ui/Shop/AccountContext.php]: ship**p**ing address

File: features/account/customer_account/viewing_orders_history/viewing_order.feature [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+        And I addressed it to "Lucifer Morningstar", "Seaside Fwy", "90802" "Los Angeles" in the "United States"
+        And for the billing address of "Mazikeen Lilim" in the "Pacific Coast Hwy", "90806" "Los Angeles", "United States"
+        And I chose "Free" shipping method with "Cash on Delivery" payment
+        And I bought a single "Angel T-Shirt"
+
+    @ui
+    Scenario: Viewing basic information about an order
+        When I view the summary of the order "#00000666"
+        And it should has number "#00000666"
+        And I should see "Lucifer Morningstar", "Seaside Fwy", "90802", "Los Angeles", "United States" as shiping address
```
Comment (tuka217) [inline features/account/customer_account/viewing_orders_history/viewing_order.feature]: ship**p**ing

**Comment by pjedrzejewski** [comment]
Thank you Grzesiu, nice work!

**Pull request merged**